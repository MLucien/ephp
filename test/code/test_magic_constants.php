<?php

function data() {
	print basename(__FILE__) . "\n";
	print basename(__DIR__) . "\n";
	print __LINE__ . "\n";
	print __FUNCTION__ . "\n";
}

data();
print __FUNCTION__ . "\n";

