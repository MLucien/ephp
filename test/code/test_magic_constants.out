test_magic_constants.php
code
6
data

